plugin/ocp_get_feature_fid_c3h:Added the OCP Get Feature FID=C3h comm…

Gopinath511 opened the pull request:
…and api

Enabled Get Feature command (FID=C3h) api with sel, namespace-id, no-uuid command line arguments. namespace-id added to the test the command with active, inactive and invalid nsid values.

Reviewed-by: Karthik Balan <karthik.b82@samsung.com>
Reviewed-by: Arunpandian J <arun.j@samsung.com>

Gopinath511 commented:
Hi @ikegami-t , Please review this PR

ikegami-t commented:
Let me confirm again as confirmed for the PR: #2768 before.
As same with the OCP feature C5h the command feature described as below but still is this command really required to add?
```
A Get Features command for Feature Identifier C3h should not be issued because no useful information is
returned.
```
I just thought only for the testing purpose we can use the existing `get-feature' command. How do you think about this?

igaw commented:
Asides @ikegami-t question, this needs to be rebased.

Gopinath511 commented:
> Let me confirm again as confirmed for the PR: #2768 before. As same with the OCP feature C5h the command feature described as below but still is this command really required to add?
> 
> ```
> A Get Features command for Feature Identifier C3h should not be issued because no useful information is
> returned.
> ```
> 
> I just thought only for the testing purpose we can use the existing `get-feature` command. How do you think about this?
> 
> (**Added**) My concern is the command to add seems not for generic purpose but only for your testing so still I can not understand the reason to add.
> 
> (**Added**) The PR changes themselves seem basically okay but still some minor comments. But before the comments let me confirm above questions to make sure.

@ikegami-t  - We have added the get feature FID=C3h command api to test the command with various NSID value and also ocp-nvme.c library should have set feature and get feature command api for all the FIDs

ikegami-t commented:
Okay (but I do not think the command added by this PR not used actually (basically)).
By the as mentioned by the comment: https://github.com/linux-nvme/nvme-cli/pull/2788#pullrequestreview-2813374513 the commit message should be fixed for this PR.
> 2. For the commit message seems needed to be followed the following guide line.
>    https://docs.kernel.org/process/submitting-patches.html#subject-line

igaw commented:
Didn't even build, fixed the obvious problem and also updated the commit message to follow the basic contributions rules. @Gopinath511 please take a look and try to follow them for the next time. Thanks!

Gopinath511 commented:
> Didn't even build, fixed the obvious problem and also updated the commit message to follow the basic contributions rules. @Gopinath511 please take a look and try to follow them for the next time. Thanks!

Sure. Thank you
